ui/styles/src/base/_form.scss

```diff
@@ -85,6 +85,10 @@ div.pods-textfield input,
 }
 
 /* textarea/wysiwyg */
+.pods-dfv-field .wp-editor-tools {
+	z-index: auto;
```

Comment:
My only thought about this would be to check the select field and adjust the z-index to pull it up higher than the WYSIWYG; instead of pulling the WYSIWYG editor down.

I looked into the select field a bit, but couldn't track down exactly where that CSS is coming from. It seems like its loading from JS somewhere, but I can't find it.

Reply:
@nicdford Check the DFV src folder for the field type specific JS, there are some embedded styles there. The aim was to keep those newer things tied directly to each field type for organization purposes.

Reply:
@nicdford  I wanted to do that as well but that didn't really work very well due to the layering of elements.
After trial and error I didn't really understand why the editor had a z-index in the first place so I simply removed it :)

Reply:
Yeah, I'm not sure about that either. It looks like it's coming from WP core from what I can tell. I was able to find the react-select component though and add a style there if you agree that method is less intrusive, I'd say we run with that.